### PR TITLE
Feat 620 usage workflow

### PR DESCRIPTION
**Pull request by irina060981**
Implementing a new workflow for word usage examples
https://github.com/alpheios-project/components/issues/620
https://github.com/alpheios-project/components/issues/632

add styling to textWork in source
For the issue - https://github.com/alpheios-project/components/issues/631
image

### CODE REVIEW

File: src/vue/components/word-usage-examples/word-usage-examples-filters.vue [hunk 1]
```diff
@@ -1,61 +1,36 @@
...
-            v-if="checkVisibilityFilterOption(typeFilterItem)"
-        >
-          <input type="radio" :id="typeFilterItem.value" :value="typeFilterItem.value" v-model="typeFilter" :disabled = "typeFilterItem.disabled === true">
-          <label :for="typeFilterItem.value">{{ typeFilterItem.label }}</label>
+    <div class="alpheios-word-usage-header-filters">
+      <p class="alpheios-word-usage-get-data-progress" v-show="gettingResult">{{ l10n.getText('WORDUSAGE_GETTING_RESULT') }}</p>
+
+      <div v-show="showHeader && !collapsedHeader">      
+        <div class="alpheios-word-usage-filters-select" v-if="authorsList">
+          <p class="alpheios-word-usage-filter-title">Author focus</p>
```
Comment (balmas) [inline src/vue/components/word-usage-examples/word-usage-examples-filters.vue]: move "Author focus" to messages file

File: src/vue/components/word-usage-examples/word-usage-examples-filters.vue [hunk 2]
```diff
@@ -1,61 +1,36 @@
...
+            >
+                <option
+                    v-for="(authorItem, authorIndex) in lastAuthorsList" v-bind:key="authorIndex"
+                    v-bind:value="authorItem"
+                    :class='{ "alpheios-select-disabled-option": !authorItem}'
+                    >{{ calcTitle(authorItem, 'author') }}</option>
+          </select>
+        </div>
+        <div class="alpheios-word-usage-filters-select" v-if="filteredWorkList">
+          <p class="alpheios-word-usage-filter-title">Work focus</p>
```
Comment (balmas) [inline src/vue/components/word-usage-examples/word-usage-examples-filters.vue]: move "Work focus" to messages file

File: src/vue/components/word-usage-examples/word-usage-examples-filters.vue (hunk 1 above)
Comment (irina060981) [reply src/vue/components/word-usage-examples/word-usage-examples-filters.vue]: Fixed

File: src/vue/components/word-usage-examples/word-usage-examples-filters.vue (hunk 2 above)
Comment (irina060981) [reply src/vue/components/word-usage-examples/word-usage-examples-filters.vue]: Fixed

**Pull request merged**